arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
```

Comment:
@matthchr - yes, I understand that. And today we don't throw on properties that are nested a level deeper, but it is still not recommended. If its not a big change, my recommendation would be to change it. Else, its ok.